index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile First Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!--MAIN CONTENT-->
+    <main class="main">
+
+        <!--Header-->
+        <header>
+            <h1>John Doe</h1>
+
+            <div id="navig">
+                <label for="humberger" id="hum">&#9776;</label>
+                <input type="checkbox" id="humberger"/>
+                <div id="navitems">
+                    <a href="#">Portfolio</a>
+                    <a href="#">About</a>
+                    <a href="#">Contact</a>
+                    <a href="#"><img alt="Mail" src="/assets/mail-icon.svg" width="20" height="30"></a>
+                </div>
+                </div>
+</header>
+<!--Toolbar-->
+<section id="toolbar">
+    <div id="heytext">
+        <p>Hey There.</p>
+        <p>I’m Alex</p>
+    </div>
+    <div id="whoami">
+        <p>I’am a Software Developer</p>
+        <p>I can help you build a product , feature<br>
+            or website Look through some of my<br> work and experience! If you like what you<br>
+             see and have a project you need coded,<br> 
+        don’t hestiate to contact me.</p>
+    </div>
```

Comment:
- [ ] "I'am a Software Developer" looks like a sub-title and should be placed in a proper `<h>` tag for better semantics. Remember the hierarchy of heading tags for this. Also double-check typos before submitting a Pull Request.
 - [ ] Please try to follow the Figma design as much as possible. What I mentioned for the heading applies to this too.
 -Your design:
<img width="425" alt="image" src="https://user-images.githubusercontent.com/61919617/186483681-17e77be7-15b0-4c03-a99f-09b69bc9049d.png">
-Figma design:
<img width="552" alt="image" src="https://user-images.githubusercontent.com/61919617/186483760-5ce78a9c-9eef-4855-9ea4-f3b835e83581.png">